config.go

```diff
@@ -0,0 +1,66 @@
+// Copyright © 2024 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package activemq
+
+import "time"
+
+//go:generate paramgen -output=paramgen_src.go SourceConfig
+//go:generate paramgen -output=paramgen_dest.go DestinationConfig
+
+type Config struct {
+	// URL is the URL of the ActiveMQ classic broker.
+	URL string `json:"url" validate:"required"`
+
+	// User is the username to use when connecting to the broker.
+	User string `json:"user" validate:"required"`
+
+	// Password is the password to use when connecting to the broker.
+	Password string `json:"password" validate:"required"`
```

Comment:
Is it possible to use AMQ without authentication?

Reply:
Yup, but should I make them optional then? I have never run activemq classic on production, but I assume that even in a dev environment one would use some sort of credentials, right?

An alternative approach could be to remove the "required" validations and make the `stomp.ConnOpt.Login` option optional.

Reply:
I think it makes sense having them as optional. I agree with you, in most cases you'll still want to use credentials. However, if ActiveMQ itself allows no credentials, then we should make such use cases possible too.